packages/http-client-java/generator/http-client-generator-test/tsp/arm.tsp

```diff
@@ -261,6 +261,17 @@ model NginxConfigurationRequest {
 
 model ChildResourceListResult is Azure.Core.Page<ChildResource>;
 
+@resource("resourceLroNoBody")
+model ResourceLroNoBody is TrackedResource<ResourceLroNoBodyProperties> {
+  ...ResourceNameParameter<ResourceLroNoBody>;
+}
+
+model ResourceLroNoBodyProperties {
```

Comment:
A resource with no property seems a bit strange to me. I cannot think of a use case of a resource like this. 
I tried to search from the spec PR you provided: https://github.com/Azure/azure-rest-api-specs/pull/36556. But it seems lots of file changes and I did not find a resource with no body property. Could you please help me to locate where is the resource that defined by service team?

Reply:
No. I just didn't bother to add any property, as this does not matter in the case at hand.

And this is the properties model (and it has one `provisioningState`. The request body (and final result) would be the `ResourceLroNoBody` model.